articles/hdinsight/kafka/apache-kafka-producer-consumer-api.md

```diff
@@ -121,56 +128,14 @@ If you would like to skip this step, prebuilt jars can be downloaded from the `P
     mvn clean package
     ```
 
-    This command creates a directory named `target`, that contains a file named `kafka-producer-consumer-1.0-SNAPSHOT.jar`.
+    This command creates a directory named `target`, that contains a file named `kafka-producer-consumer-1.0-SNAPSHOT.jar`. For ESP clusters the file will be `kafka-producer-consumer-esp-1.0-SNAPSHOT.jar`
 
 3. Replace `sshuser` with the SSH user for your cluster, and replace `CLUSTERNAME` with the name of your cluster. Enter the following command to copy the `kafka-producer-consumer-1.0-SNAPSHOT.jar` file to your HDInsight cluster. When prompted enter the password for the SSH user.
 
     ```cmd
-    scp ./target/kafka-producer-consumer-1.0-SNAPSHOT.jar sshuser@CLUSTERNAME-ssh.azurehdinsight.net:kafka-producer-consumer.jar
-    ```
-
-## <a id="run"></a> Run the example

```

Comment:
@anushreeringne was the deletion of the 'Run the example' section intentional? Why was the section removed?

Reply:
Nope this was not. let me add it again